src/libs/auth.ts

```diff
@@ -74,6 +76,8 @@ export const signOut = () => {
 };
 
 export const signOutAfterSessionTimeout = () => {
+  const user: User = getUser();
```

Comment:
This is not the correct type. See https://github.com/DataBiosphere/terra-ui/blob/73e823cd215e0d63f70f7d42a2f803523923836a/src/libs/auth.js#L237-L251 for the type of `authStore.get().user`.